README.md

```diff
@@ -121,6 +121,97 @@ get 'products' do
 end
 ```
 
+### Relational Extensions
+
+If you use either `ActiveRecord` or `Mongoid`, it is possible to represent that relationship using the `#relation` DSL method.
+
+#### Example Models
+
+```ruby
+  class Bar < ActiveRecord::Base
+    belongs_to :foo
+  end
+
+  class Foo < ActiveRecord::Base
+    has_many :bars  
+  end
+```
+
+#### Designing Representers
+
+Arguments passed to `#relation` are forwarded to `roar`. Single member relations (e.g. `belongs_to`) are represented using `property`, collections are represented using `collection`, so as such, use only arguments that apply.
+
+If `embedded` is `false`, a `_links` entry will be added. `#link_self` decorates the entity with a RESTful route to the current resource. A default base URI is constructed from a `Grape::Request` by concatenating the `#base_url` and `#script_name` properties. The resource path is extracted from the name of the relation.
+
+Otherwise, the extensions attempt to look up the correct representer module/class for the objects (e.g. we infer the `extend` argument). You can always specify the correct representer to use on your own. 
+
+
+```ruby
+class FooEntity < Grape::Roar::Decorator
+  include Roar::JSON
+  include Roar::JSON::HAL
+  include Roar::Hypermedia
+
+  include Grape::Roar::Extensions::RelationalModels
+
+  relation :belongs_to, :bar, embedded: true
+
+  link_self
+end
+
+class BarEntity < Grape::Roar::Decorator
+  include Roar::JSON
+  include Roar::JSON::HAL
+  include Roar::Hypermedia
+
+  include Grape::Roar::Extensions::RelationalModels
+
+  relation :has_many, :bars, embedded: false
+
+  link_self
+end
+```
+
+Although this example uses `Grape::Roar::Decorator`, you can also use a module as show in prior examples above. If doing so, you no longer have to mix in `Grape::Roar::Representer`. 
+
+#### Advanced Usage
+
+##### Override base URI mappings
+```ruby
+class BarEntity < Grape::Roar::Decorator
+  include Roar::JSON
+  include Roar::JSON::HAL
+  include Roar::Hypermedia
+
+  include Grape::Roar::Extensions::RelationalModels
+
+  # These are the same opts that are given to you previous examples
+  map_base_url do |opts|
```

Comment:
I think that it's more illustrative to the user to include "map"; it's more idiomatic to the process of translating the `opts` hash into a string, no?